src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -170,6 +170,11 @@ extern id const kGRPCTrailersKey;
 @property (atomic, copy, readwrite) NSString *serverName;
 
 /**
+ * The timeout for the RPC call in milliseconds. If set to 0, the call will not timeout.
```

Comment:
This sounds like a value of 0 means the call could hang indefinitely or similar.  Is that really what it means?

If so, that's not really what we're looking for. :-)  I'd like a way to make the call immediately attempt to connect to the backend (no backoff delay), and immediately surface an error to us if the connection fails (no retries).  That way we can do our own backoff / retry logic.

So I'd expect the option we're looking for to be something like a boolean retryOnFailures or something.